### DISCUSSION TITLE
How to claim 1.75XCH Pool reward after leaving from Pool Space

### DISCUSSION THREAD

**Discussion by bejjang2** [Chia-Network/chia-blockchain#8301] [Support]
![photo](https://user-images.githubusercontent.com/31396363/131779523-c8da6b14-01f5-47b3-944d-b062804a0636.png)
Anyon can help me how claim my self pool reward below ?
Here is story what happens.
First I joined the created nftplot to Space Pool , then I changed from Space Plool to SelfPool  1month later.
Then I got the block proof in Selfpool and got farmer reward 0.25 to my original wallet, but 1.75 for pool reward still remains on my pool Contract address.
But for the Claim  Reward , the GUI didn’t show the Claim Button as you see below photo.
![xchphoto](https://user-images.githubusercontent.com/31396363/131779571-44389592-ada3-43d3-8414-d639ca264b6c.png)

How do I claim of this ?

**Comment by loppefaaret** [comment]
log into the chia app with the key that controls the plotNFT that won the rewards - goto the "Pool" tab, and click the "Claim Rewards" button on the plotNFT - that will initiate a transaction from the plotNFT pool wallet, into the regular wallet

**Reply by bejjang2** [reply]
But  in "Pool" tab , there is no " Claim Rewards" button. 
![KakaoTalk_20210902_110902074](https://user-images.githubusercontent.com/31396363/131929115-dae43924-72ef-4bf4-97cd-8291d3ea1222.png)

**Reply by loppefaaret** [reply]
![image](https://user-images.githubusercontent.com/32275244/132015497-986972de-75db-4c44-9e1b-6d9a38e7caaa.png)
load the correct key!

**Comment by BrandtH22** [comment]
@bejjang2 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
